public/index.html

```diff
@@ -150,6 +151,21 @@
       }
       #root {
         height: 100%;
+        -webkit-touch-callout: none;
+        -webkit-user-select: none;
+        -khtml-user-select: none;
+        -moz-user-select: none;
+        -ms-user-select: none;
+        user-select: none;
+
+        @media screen and (min-device-width: 1200px) {
+          -webkit-touch-callout: default;
+          -webkit-user-select: auto;
+          -khtml-user-select: auto;
+          -moz-user-select: auto;
+          -ms-user-select: auto;
+          user-select: auto;
+        }
```

Comment:
I'm not sure about this one as it will likely affect cases where we want to allow selection.

If we do end up doing this, let's not use `min-device-width` which is [deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/device-width), and use `min-width` instead. But I'd first like to know what exact thing is being incorrectly selected during contextMenu open and disable the `user-select` there (or fix it with other means).

Reply:
Hey @dwelle,
I see what you mean, also thanks for spotting the `min-device-width`, I'll fix that asap.